CIScripts/Common/Invoke-CommandWithFunctions.ps1

```diff
@@ -39,19 +36,12 @@ function Invoke-CommandWithFunctions {
     }
 
     try {
-        $Output = Invoke-Command -Session $Session -ScriptBlock $ScriptBlock
+        Invoke-Command -Session $Session -ScriptBlock $ScriptBlock
```

Comment:
This *is* the last line of the function, modulo the finally block.

> Also, someone may add more lines after finally block, and change the behaviour of the function by accident.

It won't pass the tests then.

Also, this behaviour is more consistent with regular `Invoke-Command`. Assuming we store it in a variable and return in later, imagine the following scenario:

```powershell
Invoke-CommandWithFunctions -Functions "Some" {
    Print-SomethingToOutput # ▲
    Maybe-Throw
} | Out-Host
````

In case of exception being thrown, output of ▲ won't be printed anywhere. If you insist, I will change it though.

Perhaps adding `| Out-String` (which is no-op) in this line will make the code more readable instead?

Reply:
> This is the last line of the function, modulo the finally block.

I know it's technically the last line of the function, but it's not visually. This may lead to weird bugs. That was my point.